src/modules/storage/LedgerStorage.ts

```diff
@@ -633,6 +642,30 @@ export class LedgerStorage extends Storages
                 });
         });
     }
+    /**
+     * Store the CoinMarketcap data
+     * @param data IMarketCap
+     * @returns Returns the Promise. If it is finished successfully the `.then`
+     * of the returned Promise is called with the records
+     * and if an error occurs the `.catch` is called with an error.
+     *  
+     */
+    public storeCoinMarket(data: IMarketCap): Promise<any> {
+
+        return new Promise<void>((resolve, reject) => {
+            let sql =
+                `INSERT IGNORE INTO marketcap (last_updated_at, price, market_cap, change_24h, vol_24h) 
+            VALUES (?, ?, ?, ?, ?) 
+            `;
+
+            this.run(sql, [data.last_updated_at, data.price, data.market_cap, data.change_24h, data.vol_24h])
+                .then((result:any) => {
```

Comment:
Why don't you save the `last_updated_at` in ISO 8061 format so that it can be easily read?

Reply:
Block timestamp is in seconds and we want to keep time in seconds in mysql.